src/proto/Algorand.proto

```diff
@@ -7,23 +7,39 @@ message TransactionPay {
     string to_address = 1;
     uint64 fee = 2;
     uint64 amount = 3;
-    uint64 first_round = 4;
-    uint64 last_round = 5;
+}
+
+message AssetTransaction {
+    string to_address = 1;
+    uint64 fee = 2;
+    uint64 amount = 3;
+    uint64 asset_id = 4;
+}
+
+message AssetOptIn {
```

Comment:
What does `AssetOptIn` do?

Reply:
When you use an algorand assets you need to opt-in on this asset, before you can receive it or send, it's kind of registration on some token in network

Reply:
https://developer.algorand.org/docs/features/transactions/#opt-in-to-an-asset